### DISCUSSION TITLE
Lumina 配置文件格式

### DISCUSSION THREAD

**Discussion by MC-XiaoHei** [LeavesMC/Lumina#14] [Polls]
![348785462-d1c4a49c-941d-44a9-adbe-bbc8b74aad3b](https://github.com/user-attachments/assets/9a9badcf-b0a6-4295-9448-dee2182a8f52)
大家都知道yml和json，上面是hocon

**Comment by LittleChest** [comment]
yaml 😭😭😭😭😭😭😭😭😭😭

**Comment by s-yh-china** [comment]
折中一下，我们用json吧

**Comment by Bluemangoo** [comment]
hocon good

**Comment by Lumine1909** [comment]
分析一下票是谁投的（
小黑芒果是hocon，v姐是json很简单
我和小箱子投的yaml 但是还差一票 盲猜是服务器面包qwq

**Comment by aeroidesdev** [comment]
Yaml Yaml~~~

**Comment by sensen1234** [comment]
make yaml great again

**Comment by MC-XiaoHei** [comment]
~~经过所有CoreDev的最终商议，采用类Json风格的Hocon（
很感谢各位的投票（
以及其实hocon很灵活，支持很多风格（）
（鞠躬）~~
我最终决定尊重社区意见，默认采用Yaml，可选通过Debug参数使用Hocon或者Json风格的Hocon

**Reply by LittleChest** [reply]
> 经过所有CoreDev的最终商议，采用类Json风格的Hocon（ 很感谢各位的投票（ 以及其实hocon很灵活，支持很多风格（） （鞠躬）

😅😅😅😅😅😅

**Reply by Lumine1909** [reply]
![image](https://github.com/user-attachments/assets/c3c5b401-dc95-46cc-ad7f-785928d04602)

**Reply by sensen1234** [reply]
😅《我们永远不会期待用户对我们的期待》

**Comment by MC-XiaoHei** [comment]
目前就是默认情况下使用Yaml（什么也不做）
若指定`-Dlumina.config.style.hocon`或服务端运行目录下有`lumina.conf`则会使用Hocon（默认风格）
若指定`-Dlumina.config.style.json`则会使用Hocon（Json风格）
满意吗（

**Reply by Lumine1909** [reply]
Love you!
